### ISSUE TITLE
Feature Request: Allow src files outside project folder

### ISSUE THREAD

**Issue by RobertoSnap** [nomiclabs/buidler#739]
This has been discussed on Telegram, just adding it here.

I'm creating a tool which among other things has a generator for a Buidler setup. Would make this less hacky if Buidler could allow source outside Buidler folder.

## Error

BDLR401: Tried to import file outside your project

One of your projects tried to import a file that it's outside your Buidler project.

This is disabled for security reasons.

_Thank you again for Buidler, its the best!_

**Comment by alcuadrado** [comment]
Hey @RobertoSnap, thanks for opening this issue!

Can you give us more info about your use case? What are you trying to achieve?

**Comment by RobertoSnap** [comment]
Its a generator for a buidler project (with typechain and deploy) it installs the right packages, generating files and runs buidler .

The developer has this folder structure

/src/contracts/contractProject1/(...).sol
*/src/contracts/contractProject2/(...).sol
.cache/buidler/buidler.config.js
.cache/buidler/deploy/project1.js
.cache/buidler/deploy/project2.js
.cache/buidler/deploy/node_modules/..
etc

would be nice if i could point contracts in buidler.config.js to the src folder

**Comment by RobertoSnap** [comment]
There would maybe be a problem handling imports? Is that the why behind BDLR401?

**Comment by RobertoSnap** [comment]
I tried this for about half-hour with different path settings but could not get it to work. Usually ended up with files out of the project message. 

I can look more at this some other time. We went with a workaround which exposes the config file (can be good to start with, but we are trying to abstract it away to make it as dead simple as it can be for certain use cases). 

Thanks for your suggestion!
